app-office/planner/planner-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Project manager for Gnome"
+HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/planner.git"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64"
+fi
+
+SLOT="0"
+LICENSE="GPL-2+"
+IUSE="database-gda examples gtk-doc" # eds
+
+# deps on bottom from qa-vdb
+RDEPEND="
+	>=dev-libs/glib-2.50:2
+	>=x11-libs/gtk+-3.10:3
+	>=dev-libs/libxml2-2.6.27:2
+	>=dev-libs/libxslt-1.1.23
+	database-gda? ( >=gnome-extra/libgda-1.0 )
+
+	app-accessibility/at-spi2-core
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango
+"
+	# eds support is broken for now
+	# eds? (
+	# 	>=gnome-extra/evolution-data-server-3.6:=
+	# 	>=mail-client/evolution-3.6
+	#	>=app-text/libebook-1.2
+	# )
+DEPEND="${RDEPEND}
+	dev-util/gtk-doc
```

Comment:
gtk-doc dep should be behind gtk-doc USE flag, otherwise the USE flag defeats the purpose.
I'm likely going to rip out the gtk-doc before releases though, because the libplanner library is made private, so you might as well just hard disable it instead.

Reply:
nah lets keep it in, since it works.

Reply:
It won't work when I remove it upstream soon, as long as I remember about it. But then the meson_option will go away too, and we need to adapt ebuild anyways, so OK